examples/poyo/configs/base.yaml

```diff
@@ -16,12 +16,10 @@ num_workers: 4
 seed: 42
 
 subtask_weights:
-  - 1.0  # RANDOM
-  - 0.1  # HOLD
-  - 5.0  # REACH
-  - 1.0  # RETURN
-  - 0.1  # INVALID
-  - 0.0  # OUTLIER
+  random_period: 1.0
+  hold_period: 0.1
+  reach_period: 5.0
+  return_period: 1.0
 
```

Comment:
Note: We're missing `invalid_period`.

Reply:
I am seeing a difference in POYO-MP performance curves because of this. We should put back Invalid phase and test before merging.